ts/IterableInboxMessageCell.tsx

```diff
@@ -1,47 +1,111 @@
 "use strict"
-import React, { Component } from "react"
-import { View, Text, StyleSheet } from "react-native"
+import React, { Component, useState } from "react"
+import { 
+   View, 
+   Text, 
+   StyleSheet, 
+   TouchableOpacity 
+} from "react-native"
 
 type MessageCellProps = {
+   index: number,
    message: {[key: string]: any},
+   handleMessageSelect: Function,
    last: boolean 
 }
 
-const IterableInboxMessageCell = ({ message, last }: MessageCellProps) => {
+const IterableInboxMessageCell = ({ index, message, handleMessageSelect, last }: MessageCellProps) => {
    const unreadIndicator = "\u2022";
    const messageTitle = message.inboxMetadata.title
    const messageBody = message.inboxMetadata.subtitle
    const messageCreatedAt = message.createdAt
```

Comment:
i see that our wonderful mobile team hates the ending `;`. i am sorry you have to deal with that. 😆 .

Reply:
Thanks. I will add the semi-colons.